packages/wrangler/src/publish.ts

```diff
@@ -218,7 +218,10 @@ Update them to point to this script instead?`;
 
 export default async function publish(props: Props): Promise<void> {
 	// TODO: warn if git/hg has uncommitted changes
-	const { config, accountId } = props;
+	const { config, accountId, name } = props;
+	if (accountId && name) {
```

Comment:
Adding this call to every publish will make publish slower, fyi.

Reply:
I can not think of an alternative way to do what we need to do for checking the latest deploy source, any suggestions?

Reply:
Send the data back on one of the previous calls, we already do that for a subdomain check

Reply:
I'm not sure what you mean?
Someone can make changes and deploy from the dashboard at any point in time, especially if you are on a team someone can make a change and deploy right before you publish from the the CLI. This call is to determine if the latest deploy occured on Dashboard or Wrangler before publishing.

Reply:
> Send the data back on one of the previous calls, we already do that for a subdomain check

Oh, I think your saying the call is happening higher up before someone attempts to Publish and I can pick up the data on that call. That still brings up the question of when a change can happen in the Dashboard relative to when someone attempts to Publish.

Reply:
I don't think we are doing a request that would return this information anywhere else. The subdomain request does not. 

I doubt that this will slow things down that much. The majority of the deploy time, I think, is waiting for the actual deployment request to complete.

But I do think that we should not be making a fetch request from within the dialog.tsx file. We should make the request here and only pass the `deploySource` value to the `fromDashMessagePrompt()` function.